doc/examples.rst

```diff
@@ -187,7 +187,7 @@ ignoring ``<frozen importlib._bootstrap>`` and ``<unknown>`` files::
     # ... run your application ...
 
     snapshot = tracemalloc.take_snapshot()
-    display_top(snapshot, 10)
+    display_top(snapshot, limit=10)
```

Comment:
The doc comes from https://docs.python.org/dev/library/tracemalloc.html#pretty-top and so I would prefer to remove ", 10". "limit=10" is useless here, it's already the default value.